### PR TITLE
Add SelectPopup#Factory

### PR DESCRIPTION
**Pull request by mshin-wolvic**
This CL supports the select popup including the workaround to avoid the lost focus notification so that we keep it after showing the popup. We would revisit this after solving the widget hierarchy issue.

### CODE REVIEW

File: content/public/android/java/src/org/chromium/content/browser/input/SelectPopup.java [hunk 1]
```diff
@@ -152,7 +164,10 @@ private void show(View anchorView, long nativeSelectPopupSourceFrame, String[] i
         }
         WebContentsAccessibilityImpl wcax =
                 WebContentsAccessibilityImpl.fromWebContents(mWebContents);
-        if (DeviceFormFactor.isTablet() && !multiple && !wcax.isTouchExplorationEnabled()) {
+        if (sPopouFactory != null) {
```
Comment (zakharvoit) [inline content/public/android/java/src/org/chromium/content/browser/input/SelectPopup.java]: Typo s**Popup**Factory

File: components/embedder_support/android/java/src/org/chromium/components/embedder_support/view/ContentView.java [hunk 2]
```diff
@@ -344,7 +344,10 @@ protected void onFocusChanged(boolean gainFocus, int direction, Rect previouslyF
             super.onFocusChanged(gainFocus, direction, previouslyFocusedRect);
             if (hasValidWebContents()) {
                 getViewEventSink().setHideKeyboardOnBlur(true);
-                getViewEventSink().onViewFocusChanged(gainFocus);
+                // TODO : Remove the condition. We should fix no change the
```
Comment (zakharvoit) [inline components/embedder_support/android/java/src/org/chromium/components/embedder_support/view/ContentView.java]: nit: If I understand the problem correctly, better wording would be "We should fix the problem with losing focus after showing the popup widget.".

File: content/public/android/java/src/org/chromium/content/browser/input/SelectPopup.java (hunk 1 above)
Comment (mshin-wolvic) [reply content/public/android/java/src/org/chromium/content/browser/input/SelectPopup.java]: Done

File: components/embedder_support/android/java/src/org/chromium/components/embedder_support/view/ContentView.java (hunk 2 above)
Comment (mshin-wolvic) [reply components/embedder_support/android/java/src/org/chromium/components/embedder_support/view/ContentView.java]: Done.

**Review by jdapena — APPROVED** [review]

**Review by zakharvoit — APPROVED** [review]

**Pull request merged**